CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Guidelines for Contribution
+
+<br>
+
+## How can I contribute in vscode-seneca-college project?
+
+<br>
+
+Never contributed to open source before? Are you curious in how contributions function in our project?
+Here is a brief summary!
+
+<br>
+
+## **Here is a brief overview of how to contribute to vscode-seneca-college:**
+
+---
+
+<br>
+
+1. Start by identifying an issue or a feature that you would want to add.
+
+<br>
+
+2. In your local GitHub organization, fork the repository linked to the problem. The repository `underyour-GitHub-username/repository-name` will now be accessible to you.
+
+<br>
+
+3. Make a local copy of the repository using:
+
+```
+git clone https://github.com/github-username/vscode-seneca-college.
+```
+
+4. Create working branch:
+
+```
+git checkout -b branch-name
+```
+
+5. Include the upstream remote:
+
+```
+git push --set-upstream origin branch-name
+```
+
+6. If you think your fork is behind, pull the most recent changes from the main repository:
+
+```
+git pull upstream main
+```
+
+7. Make the necessary adjustments for the problem you're attempting to solve or the feature you want to include.
+
+<br>
+
+8. Merge the modifications into the branch you're working on by adding them to staging.
+
+<br>
+
+9. Make the changes and push them to the remote repository by:
+
+```
+git push origin branch-name-here
+```
+
+10. Send an upstream repository a **pull request**.
+
+<br>
+
+11. Include a brief description of the modifications made in the pull request's title.
+
+<br>
+
+12. Watch for a maintainer to review the pull request.
+
+<br>
+
+13. Modify the pull request if the maintaining reviewer suggests it.
+
+<br>
+
+14. After your pull request is merged, celebrate your achievement!
+
+## <br>
```

Comment:
This would render in HTML: `<h2 id=""><br></h2>` I don't think you need to make it a heading.